docs/Design/Numbers/Native Math.md

```diff
@@ -23,6 +23,49 @@ variable : bar int -7
 math absolute : bar
 ```
 
+### `math floor`
+
+`math floor` `:` `name`
+
+Generates code to compute the largest integer <= the given number.
+
+## Usage
```

Comment:
Could you actually switch the `## Usage` headings to `#### Usage`? Having fewer `#`s than the <code>### `math floor`</code> ones is an existing typo.